addons/pos_hr/views/multi_employee_sales_report.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="multi_employee_sales_report_action" model="ir.actions.report">
-        <field name="name">Sales Details</field>
+        <field name="name">Employee Sales Details</field>
         <field name="model">pos.session</field>
         <field name="report_type">qweb-pdf</field>
         <field name="report_name">pos_hr.multi_employee_sales_report</field>
-        <field name="binding_model_id" ref="pos_hr.model_pos_session"/>
-        <field name="binding_view_types">form</field>
```

Comment:
Why are you removing the these?

Reply:
The issue arises because the `binding_model_id` in the session form is set to display employee sales details. 
However, the `multi_employee_sales_report` is not compatible with printing reports from the session form view, ultimately generating a blank PDF.